js/api_openlayers.js

```diff
@@ -352,6 +352,41 @@ function displayTrack(xml, update) {
   }
 }
 
+function displayAllUsers(xml, update) {
+  altitudes = {};
+  var totalMeters = 0;
+  var totalSeconds = 0;
```

Comment:
Counting total distance, total time doesn't make sens in this case.
I am not sure about altitudes.